sources/src/main/java/com/ullink/slack/simpleslackapi/impl/SlackWebSocketSessionImpl.java

```diff
@@ -915,43 +1033,51 @@ private HttpClient getHttpClient() {
         return SlackPersona.SlackPresence.UNKNOWN;
     }
 
-    public void setPresence(SlackPersona.SlackPresence presence) {
-        if(presence == SlackPersona.SlackPresence.UNKNOWN || presence == SlackPersona.SlackPresence.ACTIVE) {
+    public void setPresence(SlackPersona.SlackPresence presence)
+    {
+        if (presence == SlackPersona.SlackPresence.UNKNOWN || presence == SlackPersona.SlackPresence.ACTIVE)
+        {
             throw new IllegalArgumentException("Presence must be either AWAY or AUTO");
         }
         HttpClient client = getHttpClient();
         HttpPost request = new HttpPost(SLACK_API_HTTPS_ROOT + SET_PERSONA_ACTIVE);
         List<NameValuePair> nameValuePairList = new ArrayList<>();
         nameValuePairList.add(new BasicNameValuePair("token", authToken));
         nameValuePairList.add(new BasicNameValuePair("presence", presence.toString().toLowerCase()));
-        try {
+        try
+        {
             request.setEntity(new UrlEncodedFormEntity(nameValuePairList, "UTF-8"));
             HttpResponse response = client.execute(request);
             String JSONResponse = consumeToString(response.getEntity().getContent());
             LOGGER.debug("JSON Response=" + JSONResponse);
-        }catch(IOException e) {
+        }
+        catch (IOException e)
+        {
             e.printStackTrace();
         }
 
     }
 
-    private long getNextMessageId() {
+    private long getNextMessageId()
+    {
         return messageId.getAndIncrement();
     }
 
     @Override
-    public void onMessage(String message) {
+    public void onMessage(String message)
+    {
         final JsonObject object = parseObject(message);
 
         LOGGER.debug("receiving from websocket " + message);
-        if ("pong".equals(object.get("type").getAsString())) {
+        if ("pong".equals(object.get("type").getAsString()))
+        {
             lastPingAck = object.get("reply_to").getAsInt();
             LOGGER.debug("pong received " + lastPingAck);
         }
-        else if ("reconnect_url".equals(object.get("type").getAsString())) {
-            webSocketConnectionURL = object.get("url").getAsString();
-            LOGGER.debug("new websocket connection received " + webSocketConnectionURL);
-        }
+        //        else if ("reconnect_url".equals(object.get("type").getAsString())) {
+        //            webSocketConnectionURL = object.get("url").getAsString();
+        //            LOGGER.debug("new websocket connection received " + webSocketConnectionURL);
+        //        }
```

Comment:
You can safely remove the commented block, git will allow to retrieve it through the history if we ever need to bring it back